modules/currencies/src/mock.rs

```diff
@@ -205,6 +205,8 @@ impl Config for Runtime {
 	type WeightInfo = ();
 	type AddressMapping = MockAddressMapping;
 	type EVMBridge = module_evm_bridge::EVMBridge<Runtime>;
+	type PaymentTransfer = ();
+	type StorageDepositFee = ConstU128<0>;
```

Comment:
unit tests needed to make sure `PaymentTransfer` and `StorageDepositFee` work as expected